PizzaBuilder/PizzaBuilder/Program.cs

```diff
@@ -0,0 +1,204 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace PizzaBuilder
+{
+    class Program
+    {
+
+        public interface IPizzaBuilder
+        {
+            void Reset();
+            void BuildDough();
+            void BuildSauce(string s = "Tomato");
+            void BuildTopping(string t = "Cheese");
+
+        }
+
+        public class WhitePizzaBuilder : IPizzaBuilder
+        {
+            private Pizza pizza = new Pizza();
+
+            public WhitePizzaBuilder()
+            {
+                this.Reset();
+            }
+
+            public void BuildDough()
+            {
+                this.pizza.addIngredient("The Best white Dough");
+            }
+
+            public void BuildSauce(string s = "Tomato")
+            {
+                this.pizza.addIngredient($"{s} Sauce");
+            }
+
+            public void BuildTopping(string t = "Cheese")
+            {
+                this.pizza.addIngredient($"{t} Toppings");
+            }
+
+            public void Reset()
+            {
+                this.pizza = new Pizza();
+            }
+
+            public Pizza getPizza()
+            {
+                Pizza cookedPizza = this.pizza;
+                this.Reset();
+                return cookedPizza;
+            }
+
+        }
+
+        public class WheatPizzaBuilder : IPizzaBuilder
+        {
+            private Pizza pizza = new Pizza();
+
+            public WheatPizzaBuilder()
+            {
+                this.Reset();
+            }
+
+            public void BuildDough()
+            {
+                this.pizza.addIngredient("The Best wheat Dough");
+            }
+
+            public void BuildSauce(string s = "Tomato")
+            {
+                this.pizza.addIngredient($"{s} Sauce");
+            }
+
+            public void BuildTopping(string t = "Cheese")
+            {
+                this.pizza.addIngredient($"{t} Toppings");
+            }
+
+            public void Reset()
+            {
+                this.pizza = new Pizza();
+            }
+
+            public Pizza getPizza()
+            {
+                Pizza cookedPizza = this.pizza;
+                this.Reset();
+                return cookedPizza;
+            }
+
+        }
+
+
+        public class Pizza
```

Comment:
Good job declaring the `pizza` class but even though your solution is correct, it should have been two classes as written in the requirements.